src/main/kotlin/org/hyacinthbots/lilybot/extensions/util/StartupHooks.kt

```diff
@@ -26,9 +31,18 @@ import org.hyacinthbots.lilybot.utils.updateDefaultPresence
  * @since 3.2.2
  */
 class StartupHooks : Extension() {
-	override val name = "startuphooks"
+	override val name = "startup-hooks"
+
+	private val scheduler = Scheduler()
+
+	private lateinit var threadTask: Task
+
+	private lateinit var guildTask: Task
 
 	override suspend fun setup() {
+		threadTask = scheduler.schedule(7.days, callback = ::cleanupThreads)
```

Comment:
If I'm reading this correctly, it'll only run if Lily doesn't restart for 7/30 days. This seems not right, and we should probably check maybe every other day.

Reply:
I was on autopilot from something else, that was meant to be 1 hour